src/test/java/org/opentripplanner/routing/algorithm/GraphPathTest.java

```diff
@@ -33,19 +36,24 @@
 public class GraphPathTest {
```

Comment:
I think it's time to finally remove this test. 

https://github.com/opentripplanner/OpenTripPlanner/issues/4105

What do you think, @hannesj ?